### PR TITLE
Add better error messages for when tarball downloads fail

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: lib/install.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ var fs = require('graceful-fs')
   , request = require('request')
   , minimatch = require('minimatch')
   , mkdir = require('mkdirp')
+  , VError = require('verror')
```
Comment (fengmk2) [inline lib/install.js:23]: No reason to include a new module.

**Comment by rvagg** [comment]
I'm inclined to not bother with this and deprecate all of the versions of pangyp in npm once node-gyp@3 goes live https://github.com/nodejs/node-gyp/pull/711 are there any major objections to that @aredridel, @fengmk2 @mzgol? ... not sure who else to ping on this.

**Comment by aredridel** [comment]
Not at all, and I've suggested a similar improvement to node-gyp.

**Comment by fengmk2** [comment]
Let's move back to node-gyp.

**Comment by mgol** [comment]
If node-gyp supports Node.js 0.10, 0.12 & 4.0 as well as io.js, this sounds good. And it will after your recent PR. :)

It's important to have such fixed node-gyp in time for Node 4.0.0, though or - IIUC - packages using pangyp, like node-sass, won't be able to compile on Node 4 and switching no node-gyp would mean dropping io.js.

This PR is not critical for pangyp anyway, #14 is.

**Pull request closed without merging**